homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -77,7 +77,8 @@ def update_bluetooth(now):
             for mac in devs_to_track:
                 _LOGGER.debug("Scanning " + mac)
                 result = bluetooth.lookup_name(mac, timeout=5)
-                if not result:
+                tempresult = bluetooth.find_service(address=mac)
+                if not result or not tempresult:
```

Comment:
Do all Bluetooth devices expose services? (Including iBeacons?)

Guess I'm ok with the idea, but would prefer a cleaner implementation of the logic:

```
if (lookup_name and 
        find_service):
   see_device
```